### PR TITLE
Footer implementation

### LINKED ISSUE #246: Create a footer
- Github repo
- K-App release version

### PR DESCRIPTION
**Pull request by kistora**
### Description of your PR

With github links :)
Will close #246 

### Pre-review TODO

- [x] PR is rebased on top of `origin/master` 
- [ ] PR provide new tests for new behaviors
- [x] PR doesn't introduce commented out code
- [x] PR has been tested locally on the main use cases
- [ ] Documentation have been updated if necessary

### CODE REVIEW

File: client/src/app/footer/footer.component.html [hunk 1]
```diff
@@ -0,0 +1,6 @@
+<div class="footer-container" fxLayoutAlign="center center">
+  <div>
+    <p class="text">K-App <a href="https://github.com/K-Fet/K-App/releases">v.1.3.0</a> | <strong>Serviam Indeclinabilem</strong> | © 2018<br />
```
Comment (Embraser01) [inline client/src/app/footer/footer.component.html]: Let's make the version dynamic (through environnement like recaptacha) with a version in dev (`v1.3.0-dev`) and one in prod `v1.3.1`

Comment (kistora) [reply client/src/app/footer/footer.component.html]: Why dev version < prod version ?
Btw, good idea about environnement !

Comment (kistora) [reply client/src/app/footer/footer.component.html]: Ah ok, nevermind !

File: client/src/environments/environment.ts [hunk 2]
```diff
@@ -5,4 +5,5 @@
 
 export const environment = {
   production: false,
+  version: 'v.1.3.0-dev',
```
Comment (Embraser01) [inline client/src/environments/environment.ts]: Yeah, actually I was thinking about loading it from package.json (root!)

**Pull request merged**